src/privateai_client/components/request_objects.py

```diff
@@ -759,6 +759,37 @@ def fromdict(cls, values: dict):
             )
 
 
+class NerTextRequest(BaseRequestObject):
+    default_link_batch = False
```

Comment:
Hey @a-guiducci , Thanks for the approval.
The NerText has most of its properties similar to ProcessText, so I have set the initializer to the same default values.
@guyd can you verify this?

Reply:
Ah I see what you're saying. Ok np. I think the default value for process text was removed at some point  but never really cleaned up :/ 

If you don't mind, let's remove the default_link_batch from the NerTextRequest because we'll never use it 🙏